src/AaveV3StrategicAssets_20230606/Core.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
+
+import {OwnableWithGuardian} from 'solidity-utils/contracts/access-control/OwnableWithGuardian.sol';
+
+abstract contract Core is OwnableWithGuardian {
+  /// @notice Provided address is 0x address
+  error Invalid0xAddress();
```

Comment:
Small gripe, but I'd rename this. 0xAddress doesn't really mean anything as 0x is just the prefix of a hex string. Maybe error ZeroAddress() or a case-by-case error.

Reply:
Renamed.